### ISSUE TITLE
`CFE_TIME_QueryResetVars` misnomer

### ISSUE THREAD

**Issue by skliper** [nasa/cFE#1543]
**Is your feature request related to a problem? Please describe.**
`CFE_TIME_QueryResetVars` actually reads in and sets variables, so "Query" isn't a great verb to use.

**Describe the solution you'd like**
Rename (maybe `RestoreFrom` or similar)

**Describe alternatives you've considered**
None

**Additional context**
Code review

**Requester Info**
Jacob Hageman - NASA/GSFC

**Comment by thnkslprpt** [comment]
Yes @skliper, I think `RestoreFromResetVars` or `RestoreFromTimeResetVars` would be suitable and have clearer meaning.
